sdk/search/search-documents/src/odata.ts

```diff
@@ -31,7 +40,11 @@ export function odata(strings: TemplateStringsArray, ...values: unknown[]): stri
   for (let i = 0; i < strings.length; i++) {
     results.push(strings[i]);
     if (i < values.length) {
-      results.push(escapeQuotesIfString(values[i], strings[i]));
+      if (values[i] == null || values[i] == undefined) {
```

Comment:
```suggestion
      if (values[i] === null || values[i] === undefined) {
```

Otherwise the first clause will always evaluate to true when the second is true.